app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,12 @@
+class FoodsController < ApplicationController 
+    def index 
+        if params[:q] != ""
+            @foods = FoodsFacade.new.get_foods_search(params[:q])
+            @search_count = FoodsFacade.new.get_foods_data(params[:q])[:totalHits]
```

Comment:
This is consuming the same API endpoint twice. We could get both of these data points with a single call (see my comment in FoodsFacade)